handbook/people-ops/hiring/evaluating_values.md

```diff
@@ -0,0 +1,29 @@
+# Evaluating values in an interview
+
+We expect every teammate, team, and our company to live up to the [Sourcegraph values](../../../company/values.md). It is therefore important taht we evaluate every candidate on the values to determin how successful they will be within Sourcegraph.
+
+It is the responsibility of every interviewer to think about these values when interviewing a candidate. When submitting feedback for a candidate, copy the list of values into your notes and score the candidate out of 5 based on the rubric.
+
+## Rubric
+
+Scored out of 5:
```

Comment:
I like that you did this in your interview feedback, but I found these scores hard to quickly digest.

What do you think of this?

```suggestion
| Score | Description |
| --- | --- |
| -1 | Exhibited behavior that is contrary to this value. |
| 0 | No evidence towards or against this value (neutral). |
| 1 | Demonstrated this value with evidence. |
| 2 | Exceptionally demonstrated this value, with abundant evidence. |
```